src/Sinch/Conversation/Messages/Message/CardMessage.cs

```diff
@@ -59,7 +59,5 @@ public override string ToString()
     public class MediaCarouselMessage
     {
         public Uri Url { get; set; }
-
-        public string Caption { get; set; }
```

Comment:
Great.
We definitively need to be able to identify such breaking change proactively